test/ringbuffer/RingbufferProxyTest.js

```diff
@@ -15,146 +15,129 @@
  */
 'use strict';
 
-const { expect } = require('chai');
```

Comment:
Why are we replacing async/await with promise chains? There are also test removals and some changes in the test logic. Are these intentional or some kind of merging error?

Reply:
That's not how it's meant to be. Looks like a rebase error. Thanks for noticing this and let me fix it.

Reply:
Fixed in deac2c149bd4aa5c0d1e4aba4c5e24917cfb62dd